### PR TITLE
UniversityofLjubljana.Orange version 3.30.2

### PR DESCRIPTION
**Pull request by Amoenus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/33949)

### CODE REVIEW

**Comment by Amoenus** [comment]
Locally it took some time but was installed silently.

File: manifests/u/UniversityofLjubljana/Orange/3.30.2/UniversityofLjubljana.Orange.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+PackageIdentifier: UniversityofLjubljana.Orange
+PackageVersion: 3.30.2
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (ItzLevvie) [inline manifests/u/UniversityofLjubljana/Orange/3.30.2/UniversityofLjubljana.Orange.installer.yaml:13]:
```suggestion
  Architecture: x64
```

File: manifests/u/UniversityofLjubljana/Orange/3.30.2/UniversityofLjubljana.Orange.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://download.biolab.si/download/files/Orange3-3.30.2-Miniconda-x86_64.exe
+  InstallerSha256: E25A87262018AB8D417D6D2B2BD423000B90B656A85D9620D613D6CEDDB1FDC6
+  InstallerSwitches:
+    Custom: /CURRENTUSER=1
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (ItzLevvie) [inline manifests/u/UniversityofLjubljana/Orange/3.30.2/UniversityofLjubljana.Orange.installer.yaml:22]:
```suggestion
  Architecture: x64
```

File: manifests/u/UniversityofLjubljana/Orange/3.30.2/UniversityofLjubljana.Orange.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+PackageName: Orange
+PackageUrl: https://orangedatamining.com
+License: GNU General Public License
+LicenseUrl: https://orangedatamining.com/license/
+ShortDescription: Open source machine learning and data visualization.
+Tags:
+- python
+- data
+- machine learning
+- data mining
```
Comment (ItzLevvie) [inline manifests/u/UniversityofLjubljana/Orange/3.30.2/UniversityofLjubljana.Orange.locale.en-US.yaml:20]:
```suggestion
- machine-learning
- data-mining
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**